### PR TITLE
Add translations for Brave News channel names

### PR DESCRIPTION
**Pull request by fallaciousreasoning**
Resolves  https://github.com/brave/brave-browser/issues/26116

## Submitter Checklist:

- [ ] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:

1. Enable the Brave News V2 flag
2. In a non-English locale, open the Brave News configure dialog
3. Channels should be translated

### CODE REVIEW

File: components/brave_new_tab_ui/components/default/braveToday/customize/SourcesListEntry.tsx [hunk 1]
```diff
@@ -86,7 +87,7 @@ export function ChannelListEntry (props: { channelId: string }) {
   const { setSubscribed } = useChannelSubscribed(props.channelId)
 
   return <Container direction="row" justify='space-between' align='center'>
-    <ChannelNameText>{props.channelId}</ChannelNameText>
+    <ChannelNameText>{getTranslatedChannelName(props.channelId)}</ChannelNameText>
```
Comment (sangwoo108) [inline components/brave_new_tab_ui/components/default/braveToday/customize/SourcesListEntry.tsx:90]: Q. Can we rename props.channelId to props.channelName? The id seems to be used as name as well but I think renaming would be good 😄

**Review by sangwoo108 — APPROVED** [review]
LGTM!

File: components/brave_new_tab_ui/components/default/braveToday/customize/SourcesListEntry.tsx (hunk 1 above)
Comment (fallaciousreasoning) [reply components/brave_new_tab_ui/components/default/braveToday/customize/SourcesListEntry.tsx]: Renamed! Mind taking another look?

**Comment by fallaciousreasoning** [comment]
I've renamed the remaining `channelId`s to `channelName`s - mind taking another look?

**Review by sangwoo108 — APPROVED** [review]
still LGTM! Thanks :)

**Comment by stephendonner** [comment]
@fallaciousreasoning just to be sure, these are what you're referring to, right?

Bildschirmfoto 2022-11-02 um 10 41 44 AM


@mkarolin will we have to wait until near-end of this release cycle, to be able to test these translations, if you know?

**Comment by fallaciousreasoning** [comment]
Yup, those are the ones :)

**Comment by stephendonner** [comment]
## Verified `PASSED` using 

Brave | 1.47.48 Chromium: 107.0.5304.91 (Build oficial) nightly (x86_64)
-- | --
Revisión | 3d5948960d62418160796d5831a4d2d7d6c90fa8-refs/branch-heads/5304@{#1097}
SO | macOS Versión 13.1 (Fase 22C5033e)

### Steps:
0. set my macOS system language to `Spanish`
1. installed `1.47.48`
2. launched Brave
3. opened `brave://flags`
4. set `brave://flags/#brave-news-v2` to `Enabled`
5. clicked `Relaunch`
6. opened a new-tab page
7. clicked on `Customize`
8. clicked on `Brave News`
9. examined the `Customize Brave News` dialog

#### Confirmed the start of translated channel names, e.g. `Crypto` -> `Criptomonedas` in the dialog

Captura de pantalla 2022-11-05 a la(s) 10 19 41 a m

**Pull request merged**